### PR TITLE
Update results as of 04 of may 2021

### LINKED ISSUE #4179: Add hardware section on JSON export

### PR DESCRIPTION
**Pull request by waghanza**
Hi, 

This `PR` update results. 

It also :

+ [x] Link custom UI, instead of having them in `README` here @SuspiciousLookingOwl 

Regards,

Closes #4179

### CODE REVIEW

**Comment by SuspiciousLookingOwl** [comment]
Hi @waghanza, any plan on adding #4179 too?

**Comment by waghanza** [comment]
done @SuspiciousLookingOwl (memory is in KB)

File: .tasks/db.rake [hunk 1]
```diff
@@ -147,6 +72,7 @@ namespace :db do
       end
     end
     data.merge!(updated_at: Time.now.utc)
+    data.merge!(hardware: {cpus: Etc.nprocessors, memory: 16282676, cpu_name: 'AMD FX-8320E Eight-Core Processor', os: Etc.uname})
```
Comment (waghanza) [inline .tasks/db.rake:75]: do not hardcode memory / cpu family

**Comment by waghanza** [comment]
fyi : seems that `guillotina` take times to build https://the-benchmarker.semaphoreci.com/jobs/6d7d433e-e47d-4b45-959b-4ec2641c0f9b @bloodbare

**Pull request merged**